content/tdd-introduction.md

```diff
@@ -0,0 +1,17 @@
+## Test Driven Development
+
+    Test-driven development (TDD) is a software development process that relies on the repetition of a very short development cycle. Requirements are turned into very specific test cases, then the software is improved to pass the new tests. This is opposed to software development that allows software to be added that isn't proven to meet requirements. Begin by writing a very small test for code that does not yet exist. Run the test and, naturally, it fails. Now write just enough code to make that test pass. Once the test passes, observe the resulting design and re-factor to remove any duplication you see. It is natural at this point to judge the design as too simple to handle all of the responsibilities this code will have.
```

Comment:
"re-factor to remove any duplication you see" replace with "re-factor as needed". As someone already mentioned there are many more refactorings than just DRY.